config/owasp/suppressions.xml

```diff
@@ -1,199 +1,18 @@
-<vulnerabilityName>CVE-2020-5408</vulnerabilityName>
-</suppress>
-<suppress>
-<notes><![CDATA[
-   file name: tomcat-embed-core-9.0.30.jar
-   ]]></notes>
-<packageUrl regex="true">^pkg:maven/org\.apache\.tomcat\.embed/tomcat\-embed\-core@.*$</packageUrl>
-<vulnerabilityName>CVE-2020-9484</vulnerabilityName>
-</suppress>
-    <suppress>
-        <notes><![CDATA[
-   file name: hibernate-core-5.4.10.Final.jar
-   ]]></notes>
-        <packageUrl regex="true">^pkg:maven/org\.hibernate/hibernate\-core@.*$</packageUrl>
-        <cpe>cpe:/a:hibernate:hibernate_orm</cpe>
-    </suppress>
-    <suppress>
-        <notes><![CDATA[
-   file name: tomcat-embed-core-9.0.30.jar
-   ]]></notes>
-        <packageUrl regex="true">^pkg:maven/org\.apache\.tomcat\.embed/tomcat\-embed\-core@.*$</packageUrl>
-        <vulnerabilityName>CVE-2020-11996</vulnerabilityName>
-    </suppress>
-    <suppress>
-        <notes><![CDATA[
-   file name: tomcat-embed-websocket-9.0.30.jar
-   ]]></notes>
-        <packageUrl regex="true">^pkg:maven/org\.apache\.tomcat\.embed/tomcat\-embed\-websocket@.*$</packageUrl>
-        <cpe>cpe:/a:apache:tomcat</cpe>
-    </suppress>
-    <suppress>
-        <notes><![CDATA[
-   file name: tomcat-embed-websocket-9.0.30.jar
-   ]]></notes>
-        <packageUrl regex="true">^pkg:maven/org\.apache\.tomcat\.embed/tomcat\-embed\-websocket@.*$</packageUrl>
-        <cpe>cpe:/a:apache_software_foundation:tomcat</cpe>
-    </suppress>
-    <suppress>
-        <notes><![CDATA[
-   file name: tomcat-embed-websocket-9.0.30.jar
-   ]]></notes>
-        <packageUrl regex="true">^pkg:maven/org\.apache\.tomcat\.embed/tomcat\-embed\-websocket@.*$</packageUrl>
-        <cpe>cpe:/a:apache_tomcat:apache_tomcat</cpe>
-    </suppress>
-    <suppress>
-        <notes><![CDATA[
-   file name: tomcat-embed-websocket-9.0.30.jar
-   ]]></notes>
-        <packageUrl regex="true">^pkg:maven/org\.apache\.tomcat\.embed/tomcat\-embed\-websocket@.*$</packageUrl>
-        <cpe>cpe:/a:java-websocket_project:java-websocket</cpe>
-    </suppress>
-    <suppress>
-        <notes><![CDATA[
-   file name: tomcat-embed-websocket-9.0.30.jar
-   ]]></notes>
-        <packageUrl regex="true">^pkg:maven/org\.apache\.tomcat\.embed/tomcat\-embed\-websocket@.*$</packageUrl>
-        <cve>CVE-2020-8022</cve>
-        <cve>CVE-2020-9484</cve>
-    </suppress>
-    <suppress>
-        <notes><![CDATA[
-   file name: postgresql-42.2.9.jar
-   ]]></notes>
-        <packageUrl regex="true">^pkg:maven/org\.postgresql/postgresql@.*$</packageUrl>
-        <cpe>cpe:/a:postgresql:postgresql</cpe>
-    </suppress>
-    <suppress>
-        <notes><![CDATA[
-   file name: postgresql-42.2.9.jar
-   ]]></notes>
-        <packageUrl regex="true">^pkg:maven/org\.postgresql/postgresql@.*$</packageUrl>
-        <cpe>cpe:/a:postgresql:postgresql_jdbc_driver</cpe>
+        <packageUrl regex="true">^pkg:maven/org\.apache\.sling/org\.apache\.sling\.javax\.activation@.*$</packageUrl>
```

Comment:
we can remove this. use testCompile (group: 'io.rest-assured

Reply:
We can't remove this, as it's giving me compilation error.Although, I tried  to upgrade version of rest assure jar and still getting test case failure issue.Hence, I have to suppress it.